### ISSUE TITLE
v4: `feathers generate authentication` doesn't work in .pre-1

### ISSUE THREAD

**Issue by MichaelJCole** [feathersjs/feathers#1404]
### Steps to reproduce

I was working from the documentation here:

https://crow.docs.feathersjs.com/guides/chat/authentication.html#generate-authentication

and it says to `feathers generate authentication` but it doesn't work.

```
$ feathers --version
4.0.0-pre.1

$ feathers generate authentication
events.js:183
      throw er; // Unhandled 'error' event
      ^
Error: You don't seem to have a generator with the name “feathers:authentication” installed.
...

$ feathers generate
Usage: feathers generate [type]

Options:
  -V, --version      output the version number
  -h, --help         output usage information

Commands:
  generate|g [type]  Run a generator. Type can be
  	• app - Create a new Feathers application in the current folder
  	• connection - Initialize a new database connection
  	• hook - Create a new hook
  	• middleware - Create an Express middleware
  	• service - Generate a new service
  	• plugin - Create a new Feathers plugin
  
  upgrade|u          Try to automatically upgrade to the latest Feathers version
  *
```

The suggested fix doesn't work: 
```
npm install generator-feathers:authentication
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "generator-feathers:authentication": Tags may not have any characters that encodeURIComponent encodes.
```

### Expected behavior
Tell us what should happen

### Actual behavior
Tell us what happens instead

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):
```
"dependencies": {
    "@feathersjs/configuration": "^4.0.0-pre.3",
    "@feathersjs/errors": "^4.0.0-pre.3",
    "@feathersjs/express": "^4.0.0-pre.3",
    "@feathersjs/feathers": "^4.0.0-pre.3",
    "@feathersjs/socketio": "^4.0.0-pre.3",
    "compression": "^1.7.4",
    "cors": "^2.8.5",
    "helmet": "^3.18.0",
    "serve-favicon": "^2.5.0",
    "winston": "^3.2.1"
  },
```
**NodeJS version**:  8.16
**Operating System**:  Ubuntu 
**Browser Version**:  Chrome
**Module Loader**:  Webpack

**Comment by popposoft** [comment]
Neither pre2 and pre3 works, any update on this?

**Comment by edwardsmarkf** [comment]
i asked the same question on feathers-slack recently.  so far i have been
unable to get authorization working on Crow, but thought it would be FAR
easier to resolve once i have a working example that was created using
generate.

Thank you,

Mark Edwards

**Comment by daffl** [comment]
Fixed in `@feathersjs/cli@4.0.0-pre.2`. A full working example with local and GitHub login can also be found in https://github.com/feathersjs/feathers-chat/pull/111
